packages/core/src/components/navbar/navbarHeading.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import * as classNames from "classnames";
+import * as PureRender from "pure-render-decorator";
+import * as React from "react";
+import * as Classes from "../../common/classes";
+import { IProps } from "../../common/props";
+
+// allow the empty interface so we can label it clearly in the docs
+// tslint:disable-next-line:no-empty-interface
+export interface INavbarHeadingProps extends React.HTMLProps<HTMLDivElement>, IProps {
+    // Empty
+}
+
+// this component is simple enough that tests would be purely tautological.
+/* istanbul ignore next */
```

Comment:
🤔 hmm so we ignore from code coverage but still have simple tests?
are those simple tests really not enough to pass the coverage bar?

Reply:
No tests should exist. The tests in this PR as of now are also artifacts from the cherry-pick error.